### ISSUE TITLE
Change language in app

### ISSUE THREAD

**Issue by Schoolkid1** [dessalines/jerboa#791]
- [ x] Did you check to see if this issue already exists?
- [ x] Is this only a single feature request? Do not put multiple feature requests in one issue.
- [ x] Is this a question or discussion? Don't use this, use https://lemmy.ml/c/jerboa

** Describe the feature request below **

In-app there is no option to change the language (at least I cannot find it).
Device used: Oppo A78
Software: Android 13

I want to add this option for my school assignment. 
@dessalines can you give me approval to work on this feature?
Is there a coding standard I have to take into account?

![Change language](https://github.com/dessalines/jerboa/assets/119050023/5478c551-89d9-40b9-98a7-eb2c0b1fc07d)

**Comment by beatgammit** [comment]
I never bothered getting permission and it has worked out. Just contribute good code and it'll get released.

I just wanted to say that I was hit by this just the other day and it's incredibly annoying that it's not a thing. So good on you for taking it on!

If you can do it, I'd love to be able to override it in comments and posts as well. But that's totally up to you if you want to implement it.

**Comment by MV-GH** [comment]
> If you can do it, I'd love to be able to override it in comments and posts as well. But that's totally up to you if you want to implement it.

Do you mean it should auto translate comments or have a separate language option for just posts and comments?

**Comment by beatgammit** [comment]
No, I meant be able to set the language field on individual comments or posts, even if it's not the system wide or app language.

So if I can write in Spanish and my phone is set to English, I want to be able to mark my post as Spanish manually. There seems to be no way to control what language my submissions are interpreted as.
